internal/ls/testsetup.go

```diff
@@ -0,0 +1,179 @@
+package ls
```

Comment:
Can this go in `ls_test`?

Reply:
Moved it to its own package in `testutil`. I'd have to rename the file to something like `ls_test.go` to have it be part of `ls_test`.